docs/book/src/migration/multi-group.md

```diff
@@ -33,42 +32,47 @@ Then, we'll rename `api` to `apis` to be more clear, and we'll move our
 existing APIs into a new subdirectory, "batch":
 
 ```bash
-mv api apis
 mkdir apis/batch
-mv apis/v* batch/
+mv api/* apis/batch
+# After check that all was moved successfully then, remove the old directory `api/`
+rm -rf api/ 
 ```
 
 Next, we'll need to update all the references to the old package name. For
-CronJob, that'll be `main.go` and `controllers/cronjob_controller.go`.  If
-you've added additional files to your project, you'll need to track down
+CronJob, that'll be `main.go` and `controllers/<group>/cronjob_controller.go`. Note that in the controllers
+a new directory needs to be created with the name of the group and its controllers moved to there as its references: 
+
+```bash
+mkdir controllers/batch
+mv controllers/* controllers/batch/
+```
+
+If you've added additional files to your project, you'll need to track down
 imports there as well.
 
-Finally, we'll add a new line to `PROJECT` that marks this as
-a multi-group project:
+Finally, we'll run the command which enable the multi-group layout in the project:
+
+```
+kubebuilder edit --multigroup
+```
 
+When the command `kubebuilder edit --multigroup` is executed it will add a new line 
+to `PROJECT` that marks this a multi-group project:
+                                                      
 ```yaml
 version: "2"
 domain: tutorial.kubebuilder.io
 repo: tutorial.kubebuilder.io/project
 multigroup: true
 ```
 
-While this option doesn't do anything right now, it'll indicate to
-KubeBuilder in future versions that this is a multi-group project.
-
-Now that we've migrated this to a multi-group project, normal KubeBuilder
-scaffolding won't work (as of KubeBuilder v2.0.0 -- see the warning at the
-top of this page).  You'll need to copy the setup that KubeBuilder
-normally does to add new groups for the moment:
-
-- Copy over `<kind>_types.go` and `groupversion_info.go` to
-  `apis/<group>/<version>`
-
-- Register the new group-version with the Scheme in `main.go`
-
-- Add a new `<kind>_controller.go` file
+Note that this option indicates to KubeBuilder that this is a multi-group project. 
 
-- Add the controller to the manager in `main.go`
+In this way, if the project is not new and has previous APIs already implemented will be
+in the previous structure. See that with the `multigroup`the Kind API's files are
+created under `apis/<group>/<version>` instead of `api/<version>`. 
+Also, note that the controllers will be created under `pkg/controllers/<group>` instead of `pkg/controllers`. 
+So, will be required to manually move just the old structures and then, updated its references.
```

Comment:
Hi @Adirio, 

Thank you for your suggestion. However, I prefer to keep the same structure/text info as it was before with the changes instead of your suggestion. I'd like to leave it for the project maintainers check.  

IMO, this conclusion makes more clear the changes in the layout for the users than just leave the tutorial steps only.

Reply:
Hi @mengqiy could you check the final text info and see if you think that more improvements or changes still required or if we could move forward with?

Reply:
Hi @camilamacedo86 
In that case, the english needs to be corrected.

> In this way, if the project is not new and has previous APIs already implemented will be in the previous structure.

I can't even understand what you are trying to say with the first sentence.

> ~See~ **Notice** that, **(comma inserted)** ~with the `multigroup`~ **in a multigroup-enabled project, (comma inserted)** the Kind API's files are created under `apis/<group>/<version>` instead of `api/<version>`. Also, note that the controllers will be created under `pkg/controllers/<group>` instead of `pkg/controllers`. ~So, will be required to manually move just the old structures and then, updated its references.~ That is the reason why we moved the previously generated APIs with the provided scripts in the previous steps. Remember to update the references afterwards.

I added parethenthesis with "comma inserted" where I added a comma as they are hard to notice otherwise even if they are in bold font. The rest of the changes are self-explanatory.

Reply:
HI @Adirio,  

Your new suggestion was accepted and addressed. 
Could you please close as solved all your comments addressed in order to make easier the review for others?

Reply:
Closing since you marked as solved.